cishouseholds/pipeline/timestamp_map.py

```diff
@@ -20,3 +20,8 @@
     "yyyy-MM-dd HH:mm:ss": ["blood_sample_collected_datetime"],
     "yyyy-MM-dd": ["blood_sample_arrayed_date", "antibody_test_result_recorded_date", "blood_sample_received_date"],
 }
+
+historic_blood_datetime_map = {
+    "yyyy-MM-dd HH:mm:ss": ["arrayed_ox_date"],
```

Comment:
These are using the stata names, so need updating to use those renamed to above to correctly parse to timestamp

Reply:
modified to pipeline names